normalizer/eval_utils.py

```diff
@@ -19,7 +20,14 @@ def read_manifest(manifest_path: str):
 
 
 def write_manifest(
-    references: list, transcriptions: list, model_id: str, dataset_path: str, dataset_name: str, split: str
+    references: list,
+    transcriptions: list,
+    model_id: str,
+    dataset_path: str,
+    dataset_name: str,
+    split: str,
+    audio_length: list = None,
```

Comment:
Passing a list of sample audio lengths and sample transcriptions times is now sufficient to compute the RTF over the short-form datasets